torch/csrc/cuda/Module.cpp

```diff
@@ -1034,8 +1034,6 @@ std::string uuid_to_string(const char* uuid_bytes) {
 static void registerCudaDeviceProperties(PyObject* module) {
   // Add _cudaDevicePropertires class to torch._C
   auto m = py::handle(module).cast<py::module>();
-  // until internal build is using a rocm version with uuid attr
-#ifndef FBCODE_CAFFE2
```

Comment:
@jeffdaily I don't know if version guard matters here so just want to check (iirc it needs to be >6.0). Internallly we're ok.

Reply:
If you're ok we're okay.  The version guard was supposed to be >= 6.0 but you internally had a frankenbuild of ROCm that claimed to be 6.0 but wasn't.  At this point we're at 6.3 in our CI so no version guard needed.